### PR TITLE
Recipes for Aloe and Gherkin

### PR DESCRIPTION
**Pull request by mariusvniekerk**

### CODE REVIEW

**Comment by mariusvniekerk** [comment]
@jakirkham ready for review

**Comment by mariusvniekerk** [comment]
@synapticarbors ready for review.

File: recipes/aloe/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    - repoze.lru
+
+test:
+  imports:
+    - aloe
+
+about:
+  home: https://github.com/aloetesting/aloe
+  license: GPL-3.0
+  license_file: '{{ environ["RECIPE_DIR"] }}/COPYING'
```
Comment (jakirkham) [inline recipes/aloe/meta.yaml]: Please ask upstream to include this in the `MANIFEST.in` and xref the issue/PR in a comment in the `meta.yaml`. Might also nudge them to fill out the license file with the full license.

File: recipes/gherkin-official/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    - python
+    - nose
+
+test:
+  imports:
+    - gherkin
+
+about:
+  home: https://github.com/cucumber/gherkin-python
+  license: MIT
```
Comment (jakirkham) [inline recipes/gherkin-official/meta.yaml]: Please ask upstream to include this in the `MANIFEST` and xref the issue/PR in a comment in the `meta.yaml`.

File: recipes/gherkin-official/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256sum }}
+
+build:
+  number: 0
+  #script: python setup.py install --single-version-externally-managed --record=record.txt
```
Comment (jakirkham) [inline recipes/gherkin-official/meta.yaml]: Should this be dropped?

File: recipes/gherkin-official/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+  script:
+    - rm gherkin/count_symbols_py2.py  # [py3k][not win]
+    - del gherkin\count_symbols_py2.py  # [py3k][win]
+    - python setup.py install
```
Comment (jakirkham) [inline recipes/gherkin-official/meta.yaml]: Needs the arguments `--single-version-externally-managed --record=record.txt` as it is using `setuptools`.

File: recipes/gherkin-official/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256sum }}
+
+build:
+  number: 0
+  #script: python setup.py install --single-version-externally-managed --record=record.txt
+  script:
+    - rm gherkin/count_symbols_py2.py  # [py3k][not win]
+    - del gherkin\count_symbols_py2.py  # [py3k][win]
```
Comment (jakirkham) [inline recipes/gherkin-official/meta.yaml]: Normally one does `# [py3k and win]` or similar in these cases.

File: recipes/gherkin-official/meta.yaml (hunk 2 above)
Comment (mariusvniekerk) [inline recipes/gherkin-official/meta.yaml]: done

**Pull request merged**